ddtrace/internal/_encoding.pyx

```diff
@@ -722,9 +734,16 @@ cdef class MsgpackEncoderV05(MsgpackEncoderBase):
                     PyLong_FromLong(<long> self.get_buffer()),
                     <Py_ssize_t> super(MsgpackEncoderV05, self).size,
                 )
-                return self._st.flush()
+                v05bytes = self._st.flush()
+                self._verify_encoding(v05bytes)
```

Comment:
AFAICT there is no `msgpack` guaranteed to be installed here (it's not a dependency of ddtrace) which means `flush` is broken.